reporting-app/app/models/oscer_task.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class OscerTask < Strata::Task
```

Comment:
I think I have a slight preference for extending the `Strata::Task` class like we had discussed before (https://github.com/navapbc/oscer/compare/main...giverm/test-scope), rather than creating an additional class between `Strata::Task` and `ReviewActivityReportTask`/`ReviewExemptionTask`

I'm open to either though. Not sure what others think